pkg/services/accesscontrol/roles.go

```diff
@@ -79,8 +84,9 @@ var usersAdminReadRole = RoleDTO{
 }
 
 var usersAdminEditRole = RoleDTO{
-	Name:    usersAdminEdit,
-	Version: 1,
+	Name:        usersAdminEdit,
+	Version:     2,
+	Description: "Allows every read action for users and in addition allows to administer users.",
```

Comment:
```suggestion
	Description: "Allows every read action for users and in addition allows to administer users.",
```
Allows [what? who ... to?] ... allows [what? who?]